<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNovatech425AWidget</class>
 <widget class="QWidget" name="GNovatech425AWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GNovatech425AWidget</string>
  </property>
  <layout class="QVBoxLayout" name="pLay">
   <item>
    <layout class="QHBoxLayout" name="portNameLayout">
     <item>
      <layout class="QVBoxLayout" name="pLEDLayout"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port Name:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="clockStatusLayout">
     <item>
      <layout class="QVBoxLayout" name="pClockLEDLayout"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Clock OK? (&quot;red&quot; means EXTERNAL clock UNLOCKED)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pStatusEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
